### DISCUSSION TITLE
curve ligne between points for Polar Chart

### DISCUSSION THREAD

**Discussion by tinitus91** [HanSolo/charts#82] [Q&A]
Hello, I would like to know is there is a way to ask Polar chart to join points with a smooth line, progressively  changing his to module to reach next ?    also could we add text or legend onto the pane ?
Best regards

**Comment by HanSolo** [comment]
I‘m currently on vacation. Could you add a sketch of what you are looking for?

**Comment by tinitus91** [comment]
sure, I will do it
